ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -11,6 +11,9 @@
 
   const dispatch = createEventDispatcher();
 
+  // TODO(sos): remove this feature flag when we're ready
+  const showDiscovery = true;
```

Comment:
We could expose `process.env.NODE_ENV === "development"` to the renderer via `native/preload.js` and show "preview-features" only when the app is started in dev mode.

Reply:
Introducing a vital configuration dimension like feature flags like this will not set us up to properly address the need for it. If we feel the urgent need for it we should spend time to do proper requirement engineering and build up a solution.

Yet I'd argue we don't need to feature guard this at the moment, we don't have a diverse audience which could be confused by this.

Reply:
Another question in this context similar to the ongoing discussions around capabilities, permissions and validations is where should the source of truth live for this.

Reply:
Let's not overcomplicate things; sounds like feature flags aren't necessary at this stage.